.github/actions/install-deps.yaml

```diff
@@ -0,0 +1,38 @@
+# This file is part of Edgehog.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# yaml-language-server: $schema=https://www.schemastore.org/github-action.json
+
+name: Install dependencies
+description: Install the system dependencies
+inputs: {}
+outputs: {}
+runs:
+  using: composite
+  steps:
+    - name: Install protobuf
+      shell: bash
+      env:
+        PB_REL: "https://github.com/protocolbuffers/protobuf/releases"
+        PB_VERSION: "30.2"
```

Comment:
In the [Astarte message hub proto](https://github.com/astarte-platform/astarte-message-hub-proto) we are on version 29.3
If there is no specific reason to move to 30.2 I would stay aligned with the other project for simplicity.

Reply:
I think we should use the latest for new development and update the one on the Astarte Message Hub.